src/audiences/ui/list.js

```diff
@@ -169,7 +175,10 @@ class List extends Component {
 	 */
 	onNextPage = () => {
 		const { page, search } = this.state;
-		this.props.onGetPosts( page + 1, search );
+		this.props.onGetPosts( {
+			page: page + 1,
+			search: search,
```

Comment:
Presumably we need the same context and status checks parameters here
```suggestion
			search: search,
			context: this.props.canCreate ? 'edit' : 'view',
			status: this.props.canCreate ? 'publish,draft' : 'publish',
```